packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/expression/output.js

```diff
@@ -1,34 +1,29 @@
-var foo =
-/*#__PURE__*/
-function () {
-  var _ref = babelHelpers.asyncToGenerator(function* () {
-    var wat = yield bar();
-  });
+var foo = function () {
+  return _wrapped.apply(this, arguments);
+};
 
-  return function foo() {
-    return _ref.apply(this, arguments);
-  };
-}();
+function _wrapped3() {
+  _wrapped3 = babelHelpers.asyncToGenerator(function* () {
+    var wat = yield foo();
+  });
+  return _wrapped3.apply(this, arguments);
+}
 
-var foo2 =
-/*#__PURE__*/
-function () {
-  var _ref2 = babelHelpers.asyncToGenerator(function* () {
+var foo2 = function () {
+  return _wrapped2.apply(this, arguments);
+},
+    bar = function () {
+  return _wrapped3.apply(this, arguments);
+},
+    function _wrapped2() {
```

Comment:
This doesn’t look correct to me. "_wrapped2" is a function expression whose value is discarded (it is not assigned to anything, only "_wrapped" is assigned to "bar". "_wrapped2" is not accessible from the function assigned to "foo2".

Reply:
good catch, it seems that `_wrapped2` and `_wrapped` were supposed to be declarations - like `_wrapped3`